api/policy.go

```diff
@@ -27,10 +27,14 @@ const (
 	// PolicyTypeSQLReview is the sql review policy type.
 	PolicyTypeSQLReview PolicyType = "bb.policy.sql-review"
 
-	// PipelineApprovalValueManualNever means the pipeline will automatically be approved without user intervention.
+	// PipelineApprovalValueManualNever means the pipeline proceeds automatically if possible without the assignee's intervention, and the assignee is chosen from the workspace owners or DBAs.
 	PipelineApprovalValueManualNever PipelineApprovalValue = "MANUAL_APPROVAL_NEVER"
-	// PipelineApprovalValueManualAlways means the pipeline should be manually approved by user to proceed.
+	// PipelineApprovalValueManualAlways means the pipeline should be manually approved by the assignee to proceed, and the assignee is chosen from the workspace owners or DBAs.
 	PipelineApprovalValueManualAlways PipelineApprovalValue = "MANUAL_APPROVAL_ALWAYS"
+	// PipelineApprovalValueManualNeverProjectOwner means the pipeline proceeds automatically if possible without the assignee's intervention, and the assignee is chosen from the project owners.
```

Comment:
nit: Put this near `PipelineApprovalValueManualNever` may be easier for eye-parsing 😃

Reply:
Done